docs/running.rst

```diff
@@ -1,13 +1,18 @@
 Running and Operating
 =====================
 
-PowerDNS is normally controlled via a SysV-style init.d script, often
-located in ``/etc/init.d`` or ``/etc/rc.d/init.d``. For Linux
-distributions with systemd, a service file is provided (either in the
-package or in the contrib directory of the tarball).
+On Linux, PowerDNS is controlled by a systemd service called
+``pdns.service``. The service definition file should be installed by
+the binary package, and can also be found in the tarball
+(``pdns.service.in`` template file).
```

Comment:
can you put new sentences on single lines? Over time we hope to get rid of all line breaks in the middle of sentences.

Reply:
Changed. I hope the new format is what you described?

Reply:
yes, thanks!